src/rules/no-duplicates.js

```diff
@@ -86,7 +86,8 @@ function getFix(first, rest, sourceCode) {
     !specifiers.some(specifier => specifier.importNode === node)
   )
 
-  const shouldAddDefault = getDefaultImportName(first) == null && defaultImportNames.size === 1
+  const firstDefaultSpecifier = getDefaultImportSpecifier(first)
```

Comment:
what about `import x from 'path'; import y from 'path';`? We can't just safely merge them into `x`.

Reply:
I think it's already covered in https://github.com/benmosher/eslint-plugin-import/blob/master/tests/src/rules/no-duplicates.js#L55-L57 by https://github.com/benmosher/eslint-plugin-import/blob/master/src/rules/no-duplicates.js#L53.

I may seem confusing because a single test checks two behaviors.

Reply:
Sure but i explicitly want tests for multiple defaults and separate nameds, so we can see what that does.

Many tests test multiple behaviors; the important thing is that we have as many tests as possible :-)

Reply:
Ok, I'll try to find some unimplemented edge cases, but I think I will have to refactor some things (if not everything) in order to fix them